ya_httpx_client/cluster.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import uuid
+from typing import TYPE_CHECKING
+
+from .service_base import ServiceBase
+
+if TYPE_CHECKING:
+    from typing import Callable, Union, SupportsInt, List, Type, Optional
+    from yapapi_service_manager import ServiceManager
+    from yapapi import WorkContext
+
+
+class Cluster:
+    # pylint: disable=too-many-instance-attributes
+    '''
+    Q: Here is a Cluster, and we have a `yapapi.Cluster`. Both seem to wrap a bunch of services.
```

Comment:
I'm not sure what `Cluster` is referred here. 

```suggestion
    Q: Here is a Cluster class, and we have a `yapapi.Cluster`. Both seem to wrap a bunch of services.
```